<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lineConnect">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>No connection</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="comL"/>
     </item>
     <item>
      <widget class="QPushButton" name="openB">
       <property name="text">
        <string>OPEN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeB">
       <property name="text">
        <string>CLOSE</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lineSun">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Set surveillance to sun</string>
    </property>
   </widget>
   <widget class="QLabel" name="sunProgress">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="textLat"/>
       </item>
       <item>
        <widget class="QPushButton" name="sendB_lat">
         <property name="text">
          <string>SEND</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="textLong"/>
       </item>
       <item>
        <widget class="QPushButton" name="sendB_long">
         <property name="text">
          <string>SEND</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="lineLat">
       <property name="text">
        <string>write yours latitude</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lineLong">
       <property name="text">
        <string>write yours longitude</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="setSun">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SET</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setInit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SET</string>
    </property>
   </widget>
   <widget class="QLabel" name="lineInit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Set inital position</string>
    </property>
   </widget>
   <widget class="QLabel" name="initProgress">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>271</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="lineAzimuth">
       <property name="text">
        <string>write azimuth</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lineElevation">
       <property name="text">
        <string>write elevation</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLineEdit" name="textAzimuth"/>
       </item>
       <item>
        <widget class="QPushButton" name="sendB_Azimuth">
         <property name="text">
          <string>SEND</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLineEdit" name="textElevation"/>
       </item>
       <item>
        <widget class="QPushButton" name="sendB_Elevation">
         <property name="text">
          <string>SEND</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="setAngles">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SET</string>
    </property>
   </widget>
   <widget class="QLabel" name="lineAngles">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Set angles</string>
    </property>
   </widget>
   <widget class="QLabel" name="anglesProgress">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>450</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>531</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>531</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>531</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="setAlarm">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red</string>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QLabel" name="lineES">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>EMERGENSY STOP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setContinue">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Continue</string>
    </property>
   </widget>
   <widget class="QLabel" name="lineContinue">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Continue work</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setStopsun">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="lineStopsun">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop surveillance</string>
    </property>
   </widget>
   <widget class="QLabel" name="sunStop">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>270</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>-11</x>
      <y>11</y>
      <width>41</width>
      <height>511</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>31</width>
      <height>511</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>531</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbSun">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>270</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set this block</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbAngles">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>450</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set this block</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
